packages/core/src/drawer/drawer.stories.ts

```diff
@@ -0,0 +1,89 @@
+import type { Drawer } from './drawer.js';
+import type { StoryObj } from '@storybook/web-components';
+import { html } from 'lit';
+import '../button/register.js';
+import './register.js';
+
+export default {
+  title: 'Drawer',
+  args: {
+    attachment: 'right',
+    buttonSize: 'sm'
+  },
+  argTypes: {
+    attachment: {
+      control: 'radio',
+      options: ['right', 'left', 'top', 'bottom']
+    },
+    buttonSize: {
+      control: 'radio',
+      options: ['sm', 'md', 'lg']
+    }
+  }
+};
+
+const onClick = (event: Event & { target: HTMLElement }): void => {
+  (event.target.nextElementSibling as Drawer).showModal();
+};
+
+export const API: StoryObj = {
+  render: ({ attachment, buttonSize }) => html`
+    <sl-button @click=${onClick}>Show Drawer</sl-button>
+    <sl-drawer .attachment=${attachment} .closeButtonSize=${buttonSize}>
+      <h2 slot="title">In this sidepanel you can find a lot of info</h2>
```

Comment:
Can this also use a `<div slot="title">`? Often using an `<h2>` automatically applies styling? And that may conflict with the drawer styling?

Reply:
But isn't this the freedom we want to give users so they can put in the drawer what they like? We just provide the slot for a title. Whether that needs a heading it is up to the context.

Reply:
Yeah, you're right :)